modules/gdscript/gdscript_parser.cpp

```diff
@@ -515,13 +515,13 @@ GDScriptParser::Node *GDScriptParser::_parse_expression(Node *p_parent, bool p_s
 			}
 
 			if (tokenizer->get_token() != GDScriptTokenizer::TK_PARENTHESIS_CLOSE) {
-				_set_error("Expected ')' after 'preload' path");
+				_set_error("Expected \")\" after \"preload\" path.");
 				return NULL;
 			}
 
 			Ref<GDScript> gds = res;
 			if (gds.is_valid() && !gds->is_valid()) {
-				_set_error("Couldn't fully preload the script, possible cyclic reference or compilation error. Use \"load()\" instead if a cyclic reference is intended.");
+				_set_error("Couldn\"t fully preload the script, possible cyclic reference or compilation error. Use \"load()\" instead if a cyclic reference is intended.");
```

Comment:
"Couldn " t"?

Reply:
Sorry, I used an automatic replacement tool and missed some errors it introduced :slightly_smiling_face:

I fixed them.